guides/getting_started.md

```diff
@@ -141,7 +144,8 @@ If a component or system is not included, it is not deployed, even if it is plac
       "name": "IncrementSystem",
       "writeAccess": ["CounterComponent"]
     }
-  ]
+  ],
+  "libs": [{ "name": "LibInitData", "func": "init" }]
```

Comment:
It'd be nice to change `libs` to `libraries` to match the directory name (and the verbosity of `components` and `systems`) above. And maybe `func` -> `function` or just ~always default to `init` and make it not configurable?

Another thought is an `initializers` key to make it clear what they're for:
```json
{
  "initializers": ["LibInitData"]
}
```